src/styles/main.css

```diff
@@ -0,0 +1,189 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  transition: all 300ms ease-in-out;
+  border: solid 1px #f3f3f3;
+  width: 200px;
+  height: 408px;
+  background-image: url("../images/imac.jpeg");
```

Comment:
Ensure that the image '../images/imac.jpeg' is correctly placed in the specified path to avoid broken image links.